.github/workflows/build.sh

```diff
@@ -78,30 +91,50 @@ case "$1" in
         fi
         export CXXFLAGS="$CFLAGS"
 
+        prefix="/usr"
+        MAKE="make"
```

Comment:
Given that `MAKE` is set in `build.yml` I think it would be better to set it at the top with
```sh
export MAKE=${MAKE:-make}
```
drop it here, and change all the places in `build.sh` where make is called. It should make it all consistent and it would be easier to move things like
```sh
(cd radamsa && make -j"$(nproc)" && make install)
```
to FreeBSD eventually to run the smoke tests.